src/server/api/schema.js

```diff
@@ -1191,14 +1191,17 @@ const rootMutations = {
         contact.campaign_id
       );
       const { assignmentId, reason } = optOut;
+      const organization = await Organization.get(campaign.organization_id);
```

Comment:
This should be `await loaders.organization.load(campaign.organization_id)` so it can use cache. (Most mutations should not use loaders in case they use stale data, but in this case, campaign is already loaded that way, and there's no such danger)